rock-paper-scissors/service/src/main.rs

```diff
@@ -86,14 +95,11 @@ impl RockPaperScissors {
         let p_move = self.player_move.as_ref().unwrap();
         let c_move = self.challenger_move.as_ref().unwrap();
         let result = p_move.compare(&c_move);
-        if result == 0 {
-            return Ok(format!("{} played {:?} and {} played {:?}. Tie!", self.player_name, p_move, self.challenger_name, c_move));
-        }
-        if result == 1 {
-            return Ok(format!("{} played {:?} and {} played {:?}. Player Wins!", self.player_name, p_move, self.challenger_name, c_move));
+        match result {
+            Ordering::Greater => return Ok(format!("{} played {:?} and {} played {:?}. {} Wins!", self.player_name, p_move, self.challenger_name, c_move, self.player_name)),
+            Ordering::Equal => return Ok(format!("{} played {:?} and {} played {:?}. Tie!", self.player_name, p_move, self.challenger_name, c_move)),
+            Ordering::Less => return Ok(format!("{} played {:?} and {} played {:?}. {} Wins!", self.player_name, p_move, self.challenger_name, c_move, self.challenger_name)),
```

Comment:
please run `cargo fmt` and `cargo clippy` on your code. CI should do this, too, if you set it up